code/game/objects/items/tools/wirecutters.dm

```diff
@@ -78,6 +78,18 @@
 
 	random_color = FALSE
 
+/obj/item/wirecutters/syndie
+	name = "wirecutters"
+	desc = "This cuts wires."
+	icon_state = "cutters_red"

```

Comment:
you could just del this and have it pull from the colors so it's not suspicious.